website/docs/r/vapp_vm.html.markdown

```diff
@@ -114,6 +114,28 @@ resource "vcd_vapp_vm" "web2" {
   depends_on = ["vcd_vapp.web"]
 }
 
+resource "vcd_vapp_vm" "internalDiskOverride" {
```

Comment:
The example became huge. Let's split it at this point into two examples, like we have in EG:
https://www.terraform.io/docs/providers/vcd/r/edgegateway.html
This example's header section could be "Example Usage (Override Template Disk)"

Reply:
added

Reply:
Thanks! Let's just remove the network and vApp part from it to keep it focused on VM (we already have a sample with vApp above):
```
resource "vcd_network_direct" "net" {
  name             = "net"
  external_network = "corp-network"
}

resource "vcd_vapp" "web" {
  name = "web"
  depends_on = ["vcd_network_direct.net"]
}
```

Reply:
done